Makefile

```diff
@@ -240,15 +240,15 @@ registry: $(CLIENT_CERT) ## Start a docker registry with certs in the mesos mast
 genconf: start $(CONFIG_FILE) ## Run the DC/OS installer with --genconf.
 	$(RM) dcos-genconf.*.tar ## Remove tar files from previous runs;  otherwise we might skip building Docker image
 	@echo "+ Running genconf"
-	@bash $(DCOS_GENERATE_CONFIG_PATH) --genconf --offline -v
+	@export PORT=${INSTALLER_PORT}; export DCOS_INSTALLER_CONTAINER_NAME=${INSTALLER_CTR}; bash $(DCOS_GENERATE_CONFIG_PATH) --genconf --offline -v
```

Comment:
nevermind. this comment was on the wrong PR.
You'll need to rebase and resolve conflicts. I've added INSTALLER_PORT and INSTALLER_CMD to master